mythril/laser/ethereum/call.py

```diff
@@ -36,7 +36,7 @@ def get_call_parameters(global_state, dynamic_loader, with_value=False):
     callee_account = None
     call_data, call_data_type = get_call_data(global_state, meminstart, meminsz, False)
 
-    if int(callee_address, 16) >= 5:
+    if int(callee_address, 16) >= 5 or int(callee_address) == 0:
```

Comment:
as callee_address is base 16 it fails whenever a string is given which isn't base10(in our case of tests native_contracts fail).